### PR TITLE
fix(ListView): Fixed issue with hideCloseIcon not being respected

### PR DESCRIPTION
**Pull request by chalettu**
Fix for issue https://github.com/patternfly/patternfly-react/issues/277 where the hideCloseIcon property did not have an effect on expanded content.

### CODE REVIEW

**Comment by chalettu** [comment]
Screenshot of expanded content if you add hideCloseIcon=true to listviewitem.  
closeicon

File: src/components/ListView/ListViewItem.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ class ListViewItem extends React.Component {
             <ListViewGroupItemContainer
               expanded={compoundExpanded}
               onClose={hideCloseIcon ? undefined : onCloseCompoundExpand}
+              hideCloseIcon={hideCloseIcon}
```
Comment (jeff-phillips-18) [inline src/components/ListView/ListViewItem.js]: No need to pass this now.

File: src/components/ListView/ListViewItem.js [hunk 2]
```diff
@@ -95,6 +96,7 @@ class ListViewItem extends React.Component {
           <ListViewGroupItemContainer
             expanded={expanded}
             onClose={hideCloseIcon ? undefined : this.toggleExpanded}
+            hideCloseIcon={hideCloseIcon}
```
Comment (jeff-phillips-18) [inline src/components/ListView/ListViewItem.js]: No need to pass this now.

**Comment by chalettu** [comment]
@jeff-phillips-18 , I removed the parameter.  Let me know if you have any more feedback.  Thanks in advance

File: src/components/ListView/ListViewItem.js (hunk 1 above)
Comment (jeff-phillips-18) [reply src/components/ListView/ListViewItem.js]: @chalettu This one is still there but can be removed.

**Comment by jeff-phillips-18** [comment]
@chalettu Still one left 😉

**Comment by chalettu** [comment]
@jeff-phillips-18 , got it.  Removed the last one.  Good catch.

**Comment by priley86** [comment]
LGTM. thanks!

**Pull request merged**